examples/medplum-patient-intake-demo/data/core/patient-intake-questionnaire.json

```diff
@@ -0,0 +1,165 @@
+              },
+              {
+                "linkId": "gender-identity",
+                "text": "Gender Identity",
+                "type": "choice",
+                "answerValueSet": "http://hl7.org/fhir/ValueSet/administrative-gender"
+              },
+              {
+                "linkId": "sexual-orientation",
+                "text": "Sexual Orientation",
+                "type": "choice",
+                "answerValueSet": "https://example.com/sexual-orientation"
+              }
+            ]
+          },
+          {
+            "linkId": "coverage-information",
+            "text": "Coverage Information",
+            "type": "group",
+            "repeats": true,
+            "item": [
+              {
+                "linkId": "insurance-provider",
+                "text": "Insurance Provider",
+                "type": "string"
+              },
+              {
+                "linkId": "subscriber-id",
+                "text": "Subscriber ID",
+                "type": "string"
+              },
+              {
+                "linkId": "relationship-to-subscriber",
+                "text": "Relationship to Subscriber",
+                "type": "choice",
+                "answerValueSet": "http://hl7.org/fhir/ValueSet/relatedperson-relationshiptype"
+              }
+            ]
+          },
+          {
+            "linkId": "social-determinants-of-health",
+            "text": "Social Determinants of Health",
+            "type": "group",
+            "item": [
+              {
+                "linkId": "housing-status",
+                "text": "Housing Status",
+                "type": "choice",
+                "answerValueSet": "http://terminology.hl7.org/ValueSet/v3-LivingArrangement"
+              },
+              {
+                "linkId": "education-level",
+                "text": "Education Level",
+                "type": "choice",
+                "answerValueSet": "http://terminology.hl7.org/ValueSet/v3-EducationLevel"
+              },
+              {
+                "linkId": "veteran-status",
+                "text": "Veteran Status",
+                "type": "boolean"
+              }
+            ]
+          },
+          {
+            "linkId": "languages-spoken",
+            "text": "Languages Spoken",
+            "type": "choice",
+            "answerValueSet": "http://hl7.org/fhir/ValueSet/languages",
+            "repeats": true
+          },
+          {
+            "linkId": "preferred-language",
+            "text": "Preferred Language",
+            "type": "choice",
+            "answerValueSet": "http://hl7.org/fhir/ValueSet/languages"
+          },
+          {
+            "linkId": "consent-sections",
+            "text": "Consent Sections",
+            "type": "group",
+            "item": [
+              {
+                "linkId": "consent-for-treatment",
+                "text": "Consent for Treatment",
```

Comment:
For these consent questions, the important part is the text associated with them. Can you copy and modify the long text describing this consent from the example

Reply:
Sure, I'll take a look at that and open another PR with the fix